inputremapper/injection/macros/macro.py

```diff
@@ -540,12 +541,18 @@ async def task(handler: Callable):
 
         self.tasks.append(task)
 
-    def add_wait(self, time: Union[int, float]):
+    def add_wait(self, time: Union[str, int], vary: int=0):
```

Comment:
If the parameter is called `vary`, I might expect gaussian randomness with `vary` being its variance. How about renaming it to `max_time`?

I wonder if defaulting it to None instead would be more consistent with the other macro functions.